curriculum/challenges/english/10-coding-interview-prep/rosetta-code/towers-of-hanoi.md

```diff
@@ -8,13 +8,13 @@ dashedName: towers-of-hanoi
 
 # --description--
 
-Solve the [Towers of Hanoi](https://en.wikipedia.org/wiki/Towers_of_Hanoi "wp: Towers_of_Hanoi") problem.
+Solve the Towers of Hanoi problem. The number of objects will be given as the first parameter, and followed by the strings used to identify each of the stacks of objects. Create a nested array containing the list of moves, `["source", "destination"]`.
```

Comment:
```suggestion
Solve the Towers of Hanoi problem. The number of objects will be given as the first parameter, followed by the strings used to identify each stack of objects. Create a nested array containing the list of moves, `["source", "destination"]`.
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix